dist/Youtube.js

```diff
@@ -37,10 +37,10 @@ THE SOFTWARE. */
   var _isOnMobile = videojs.browser.IS_IOS || videojs.browser.IS_NATIVE_ANDROID;
```

Comment:
can you revert the changes in `dist/`? They'll get updated as part of a release.